students/JerryH/Lesson09/mailroom.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+class Donor:
+
+    def __init__(self, first, last, donations = None):
+        self.first = first
+        self.last = last
+        self.donations = donations
+
+    @property
+    def full_name(self):
+        return "{} {}".format(self.first, self.last)
+
+    @property
+    def donor_history(self):
+        return sum(self.donations)
+
+    def add_donation(self, amount):
+        return self.donations.append(amount)
+
+
+class DonorBook:
+
+    def __init__(self, donors = []):
```

Comment:
better to make default be `None` -> `def __init__(self, donors=None)`
it is dangerous to have mutable sequence as a default param, and you're already checking for `is None`